packages/help-center/src/components/index.d.ts

```diff
@@ -63,17 +63,26 @@ declare module 'calypso/state/purchases/selectors' {
 	export const getUserPurchases: ( state: unknown ) => { productSlug: string }[];
 }
 
+declare module 'calypso/state/selectors/get-current-route' {
+	export const getCurrentRoute: ( state: unknown ) => string;
+}
+
```

Comment:
I had to add these to fix build errors. I hope this is the right place to put these!